beacon_node/network/src/router.rs

```diff
@@ -474,15 +475,20 @@ impl<T: BeaconChainTypes> Router<T> {
     }
 
     fn send_beacon_processor_work(&mut self, work: BeaconWorkEvent<T>) {
+        let mut error_debounce = TimeLatch::default();
```

Comment:
This latch is created immediately before the `.elapsed()` check, so it will always return `true` and won't suppress any logs.

I think we'd need to persist the latch in `self` as a field next to `beacon_processor_send`. Then it can retain memory across calls to `send_beacon_processor_work`.